frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -78,484 +94,1220 @@ export function FeatureFlag({ id }: { id?: string } = {}): JSX.Element {
-                                            </a>
-                                        </span>
-                                    ) : undefined
+                                                Delete
+                                            </LemonButton>
+                                        )}
+                                        <LemonButton
+                                            type="primary"
+                                            data-attr="feature-flag-submit"
+                                            loading={featureFlagLoading}
+                                            htmlType="submit"
+                                        >
+                                            Save changes
+                                        </LemonButton>
+                                    </div>
                                 }
-                            >
-                                {({ value, onChange }) => (
-                                    <LemonInput
-                                        value={value}
-                                        onChange={(v) => {
-                                            if (v !== value) {
-                                                setHasKeyChanged(true)
-                                            }
-                                            onChange(v)
-                                        }}
-                                        data-attr="feature-flag-key"
-                                        className="ph-ignore-input"
-                                        autoFocus
-                                        placeholder="examples: new-landing-page, betaFeature, ab_test_1"
-                                        autoComplete="off"
-                                        autoCapitalize="off"
-                                        autoCorrect="off"
-                                        spellCheck={false}
-                                    />
-                                )}
-                            </Field>
+                            />
+                            {featureFlag.experiment_set && featureFlag.experiment_set?.length > 0 && (
+                                <AlertMessage type="warning">
+                                    This feature flag is linked to an experiment. It's recommended to only make changes
+                                    to this flag{' '}
+                                    <Link to={urls.experiment(featureFlag.experiment_set[0])}>
+                                        using the experiment creation screen.
+                                    </Link>
+                                </AlertMessage>
+                            )}
+                            <EventBufferNotice additionalInfo=", meaning it can take around 60 seconds for some flags to update for recently-identified persons" />
```

Comment:
Just noticed this:

```suggestion
                            <EventBufferNotice additionalInfo=", meaning it can take around 60 seconds for some flags to update for recently-identified persons. To sidestep this, you can choose to override server properties when requesting the feature flag." />
```